### PR TITLE
fix issue with missing load statements when --module-only is used, don't skip ready/prepare steps

### PR DESCRIPTION
**Pull request by boegel**
fix for #1274

### CODE REVIEW

File: easybuild/framework/easyblock.py [hunk 1]
```diff
@@ -1738,8 +1738,10 @@ def _skip_step(self, step, skippable):
             self.log.info("Skipping %s step (skip: %s, skipsteps: %s)", step, self.skip, self.cfg['skipsteps'])
             skip = True
 
-        # skip step when only generating module file; still run sanity check without use of force
-        elif module_only and not step in ['sanitycheck', 'module']:
+        # skip step when only generating module file
+        # * still run sanity check without use of force
+        # * always run ready & prepare step to set up toolchain + deps
+        elif module_only and not step in ['ready', 'prepare', 'sanitycheck', 'module']:
```
Comment (stdweird) [inline easybuild/framework/easyblock.py]: need modifications in the help message of module only. so move the list of steps to constants and genereate the help from it

Comment (boegel) [inline easybuild/framework/easyblock.py]: OK, will do, nice catch

**Comment by boegel** [comment]
@wpoely86: please (re)review, I've fixed Stijn's remarks, but added quite a few of extra changes by introducing constants for the step names

**Comment by stdweird** [comment]
LGTM, :+1: for switching to constants

**Comment by boegel** [comment]
Thanks for the review @stdweird!

**Pull request merged**